docs/develop/pivottables.md

```diff
@@ -0,0 +1,152 @@
+- A [PivotField](/javascript/api/office-scripts/excelscript/excelscript.pivotfield) contains multiple [PivotItems](/javascript/api/office-scripts/excelscript/excelscript.pivotitem). Each PivotItem is a unique value in the field. Think of each item as a value in the table column. Items could also be aggregated values, such as sums, if the field is being used for data.
+- The [PivotLayout](/javascript/api/office-scripts/excelscript/excelscript.pivotlayout) defines how the [PivotFields](/javascript/api/office-scripts/excelscript/excelscript.pivotfield) and [PivotItems](/javascript/api/office-scripts/excelscript/excelscript.pivotitem) are displayed.
+- [PivotFilters](/javascript/api/office-scripts/excelscript/excelscript.pivotfilters) filter data from the [PivotTable](/javascript/api/office-scripts/excelscript/excelscript.pivottable) using different criteria.
+
+Let's look at how these relationships work in practice. The following data describes fruit sales from various farms. It is the base for all the examples in this article. Use <a href="pivottable-sample.xlsx">pivottable-sample.xlsx</a> to follow along.
+
+:::image type="content" source="../images/pivottable-raw-data.png" alt-text="A collection of fruit sales of different types from different farms.":::
+
+## Create a PivotTable with fields
+
+PivotTables are created with references to existing data. Both ranges and tables can be the source for a PivotTable. They also need a place to exist in the workbook. Since the size of PivotTable is dynamic, only the upper-left corner of the destination range is specified.
+
+The following code snippet creates a PivotTable based on a range of data. The PivotTable has no hierarchies, so the data is not yet grouped in any way.
+
+```typescript
+  const dataSheet = workbook.getWorksheet("Data");
+  const pivotSheet = workbook.getWorksheet("Pivot");
+
+  const farmPivot = pivotSheet.addPivotTable(
+    "Farm Pivot", /* The name of the PivotTable. */
+    dataSheet.getUsedRange(), /* The source data range. */
+    pivotSheet.getRange("A1") /* The location to put the new PivotTable. */);
+```
+
+:::image type="content" source="../images/pivottable-empty.png" alt-text="A PivotTable named 'Farm Pivot' with no hierarchies.":::
+
+### Hierarchies and fields
+
+PivotTables are organized through hierarchies. Those hierarchies are used to pivot data when added as a specific type of hierarchy. There are four types of hierarchies.
+
+- **Row**: Displays items in horizontal rows.
+- **Column**: Displays items in vertical columns.
+- **Data**: Displays aggregates of values based on the rows and columns.
+- **Filter**: Add or removes items from the PivotTable.
+
+A PivotTable can have as many or as few of its fields assigned to these specific hierarchies. Generally, a useful PivotTable needs at least one data hierarchy to show and at least one row or column to pivot on. The following code snippet adds two row hierarchies and two data hierarchies.
```

Comment:
It's unclear to me why the PivotTable needs at least one data hierarchy. Could you expand or adjust the phrasing of "a useful PivotTable needs at least one data hierarchy to show"? To *show* what?